### PR TITLE
durable task framework added to workflow

### PR DESCRIPTION
**Pull request by vip32**

### CODE REVIEW

**Comment by vip32** [comment]
fails due to build problem:
404 https://github.com/Eastrall/Ether.Network  

not my change though...

**Comment by vip32** [comment]
fails due to build problem:
404 https://github.com/Eastrall/Ether.Network

not my change though...

**Comment by vip32** [comment]
ordering done

File: README.md [hunk 1]
```diff
@@ -772,7 +772,7 @@ Follows best practices and conventions to provide you a SOLID development experi
 
 ### Workflow
 * [CoreWF](https://github.com/dmetzgar/corewf/) - Port of Windows Workflow Foundation (WF) to .NET Core.
-* [workflow-core](https://github.com/danielgerlag/workflow-core) - Lightweight workflow engine for .NET Standard.
+* [Durable Task Framework](https://github.com/Azure/durabletask) - This framework allows users to write long running persistent workflows in C# using the async/await capabilities.* [workflow-core](https://github.com/danielgerlag/workflow-core) - Lightweight workflow engine for .NET Standard.
```
Comment (ngohungphuc) [inline README.md]: Add new line before workflow-core

Comment (ngohungphuc) [reply README.md]: Will open again if you resolve my comment

**Pull request closed without merging**